dll/win32/syssetup/wizard.c

```diff
@@ -509,6 +668,9 @@ DoWriteProductOption(PRODUCT_OPTION nOption)
         goto Error;
     }
 
+    /* Write system sound events values for Workstation */
+    DoWriteSoundEvent();
```

Comment:
You get the selected option (server/workstation) in the function argument `nOption` (isn't that obvious?) :eyes:

Reply:
Indeed, it is. Thanks. :sweat_smile:

Reply:
Fixed.